vault/identity_store_oidc_test.go

```diff
@@ -1137,7 +1137,7 @@ func TestOIDC_PeriodicFunc(t *testing.T) {
 			i := 0
 			for currentCycle <= lastCycle {
 				// sleep for a rotation period
-				time.Sleep(testSet.namedKey.RotationPeriod)
+				time.Sleep(testSet.namedKey.RotationPeriod + 50*time.Millisecond)
```

Comment:
@suprjinx Do you mind removing this change from this file and then I'll merge? Thanks!